include/mcptam/KeyFrame.h

```diff
@@ -390,7 +397,13 @@ class MultiKeyFrame
 
   void RemoveImages();
 
+  //partially copy MultiKeyFrame
+  
+  MultiKeyFrame* CopyMultiKeyFramePartial(); //returns a pointer to the partially copied MKF
+  
   // Variables
+  bool isBufferMKF;  ///am I a buffer MKF?  If so, we'll use the flag to disable things such as updating measurements to the global points list when buffering keyframe information
```

Comment:
Changed the comment in KeyFrame.h (line 405)  to:  
bool isBufferMKF;  // Determines whether the current MKF is buffered. A multi-keyframe buffer is a priority queue that stores MKFs since the last insertion time until next insertion time is detected. They are scored based on the reduction in expected entropy if they were inserted in the map.

Changed the comment in KeyFrame.cpp (line 95)  to:
if(!(mpParent->isBufferMKF)) // This keyframe is NOT part of a buffer, add measurements to global points. Otherwise, disable things such as updating measurements to the global points list when buffering keyframe information.